packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -358,6 +395,28 @@ export class ProjectActions {
 
     const [newSpec] = codeGenResults.files
 
+    const cfg = this.ctx.project.getConfig()
+
+    if (cfg) {
```

Comment:
Not sure when or where but we really need to centralize this - it's a copy paste of some really complex and important code